Zilon.Core/Zilon.Bot.Players/BotActorTaskSourceBase.cs

```diff
@@ -33,7 +40,7 @@ public IActorTask[] GetActorTasks(IActor actor)
             // TODO Лучше сразу отдавать на обработку актёров текущего игрока.
             if (actor.Owner != _player)
             {
-                return System.Array.Empty<IActorTask>();
+                throw new System.Exception();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['System.Exception' should not be thrown by user code.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5544638)